docs: add Getting Started guide for new developers(Fixes #9)

arunvijo opened the pull request:
Added a detailed Getting Started section with install/run/contribute instructions for new developers. Fixes #9.

arunvijo commented:
Hey 👋
This is my first contribution to this repo! Let me know if you'd like any changes.
Thanks for the opportunity to contribute 🚀

Altair-05 commented on GETTING_STARTED.md:
```diff
@@ -0,0 +1,108 @@
+Edit
+yarn dev
+The project should now be running at:
+👉 http://localhost:3000
```
i think it should be localhost:5000 ?

Altair-05 commented on GETTING_STARTED.md:
```diff
@@ -0,0 +1,108 @@
+
+Using npm:
+
+bash
```
what is this bash copy edit for ?

Altair-05 commented on GETTING_STARTED.md:
```diff
@@ -0,0 +1,108 @@
+
+Click the **Fork** button at the top right of the repository on GitHub.
+
+### 2. Clone Your Fork
```
mention for diff systems like mac/linux and windows

Altair-05 reviewed (changes requested):
hello @arunvijo  plz check https://github.com/Altair-05/CodeBrosPlatform/blob/main/CodeReadme.md and readme file and create correct and more detailed documentation :)
and also mention for review  :):

arunvijo commented:
@Altair-05 I've pushed additional updates addressing the previous feedback. Could you please review the latest changes? 🙏  
**Changes made**:  
- Fixed port number (3000 → 5000)  
- Removed all "bash Copy Edit" lines  
- Added OS-specific instructions

Altair-05 commented:
@arunvijo  LGTM ....
once gssoc period start i will merge it
